sms_analysis.ipynb

```diff
@@ -535,30 +622,28 @@
    "execution_count": null,
    "metadata": {
     "collapsed": false,
+    "deletable": true,
+    "editable": true,
     "scrolled": true
    },
    "outputs": [],
    "source": [
     "def get_word_summary_for_contact(contact, top_n=25):\n",
-    "    contact = contact.encode('utf-8')\n",
```

Comment:
It throws an error for me in Python 2, so create a helper method at the top of the file that looks like:

```
# To aid with Python 2.7/3 compatibility,
convert_unicode_to_str_if_needed(unicode_or_str):
  if type(unicode_or_str).__name__ == 'unicode':
    return unicode_or_str.encode('utf-8')
  return unicode_or_str
```

And use this here and below.